halibot/halibot.py

```diff
@@ -130,10 +136,7 @@ def _instantiate_modules(self):
 			self.add_module_instance(k, obj(self, conf))
 
 	def get_object(self, name):
-		# TODO priority?
-		if name in self.modules: return self.modules[name]
-		if name in self.agents: return self.agents[name]
-		return None
+		return self.objects.get(name, None)
```

Comment:
I don't remember who uses this, but we might just want to completely remove `get_object`.